Examples/Scripts/SetUpSPECIES.sh

```diff
@@ -0,0 +1,471 @@
+#! /bin/bash
+
+# Species specific config file 
+
+# Takuya Hayashi, RIKEN Japan
+# Akiko Uematsu, RIKEN, Japan
+# Yuki Hori, RIKEN Japan
+# Chad Donahue, Washington University St. Louis, USA
+# Matthew F Glasser, Washington University St. Louis, USA
+# Copyright (c) 2016-2023
+# All rights reserved.
+
+## Species specific variables
+if [[ "$SPECIES" == *Human* ]] ; then
+
+	BrainScaleFactor="1"
+	CorticalScaleFactor="1"
+
+	#PreFreeSurferPipeLineBatch.sh
+	BrainExtract="${BrainExtract:-INVIVO}"
+	BrainSize="150"              #BrainSize in mm, distance bewteen top of FOV and bottom of brain
+	Defacing="TRUE"
+	betcenter="45,55,39"         # comma separated voxel coordinates in T1wTemplate2mm
+	betradius="75"               # brain radius for bet
+	betbiasfieldcor="TRUE"
+	betfraction="0.3"            # fractional intensity threshold for bet
+	bettop2center="86"           # Distance between top of FOV and center of brain
+	betspecieslabel="0"          # bet4animal species label
+
+	StrucRes=${StrucRes:-0.7}
+	FNIRTConfig="${HCPPIPEDIR_Config}/T1_2_NHP_NNP_Human_2mm.cnf" #FNIRT 2mm T1w Config
+	TopupConfig="${HCPPIPEDIR_Config}/b02b0.cnf" #Config for topup or "NONE" if not used
+	UnwarpDir="${UnwarpDir:-z}" # z appears to be best or "NONE" if not used, read from Read.direction in Seriesinfo.txt (PA:y, AP:y-, RL:x, LR:x-, FH:z, HF:z-)
+	BiasFieldSmoothingSigma="5.0"
+	T1wTemplate="${HCPPIPEDIR_Templates}/MNI152_T1_${StrucRes}mm.nii.gz"  
+	T1wTemplateBrain="${HCPPIPEDIR_Templates}/MNI152_T1_${StrucRes}mm_brain.nii.gz" 
+	T1wTemplate2mm="${HCPPIPEDIR_Templates}/MNI152_T1_2mm"  
```

Comment:
Having "2mm" in the variable name doesn't seem like a great choice, though it is our fault for putting it in the argument name.  Consider `T1wTemplateGrayord`?